engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/UniverseSetupScreen.java

```diff
@@ -188,10 +243,118 @@ private void recursivelyAddModuleDependencies(Set<Name> modules, Name moduleName
         }
     }
 
+    /**
+     * Called whenever the user decides to create a new world.
```

Comment:
"create" -> "add" ? Might be a subtle difference or a subjective call. Adding a new entry to a list vs creating a new entry to a list.